ee/modules/600-flant-integration/images/flant-pricing/hooks/prometheus_metrics.sh

```diff
@@ -324,7 +324,9 @@ function node_os_image_metrics() {
                 "(?:(r)ed (h)at (e)nterprise (l)inux.*(7|8))",
                 "(?:(centos).*(7|8))",
                 "(?:(debian).*(9|10|11))",
-                "(?:(ubuntu).*(16.04|18.04|20.04|22.04))"
+                "(?:(ubuntu).*(16.04|18.04|20.04|22.04))",
+                "(?:(rocky).*(8|9))",
+                "(?:(astra))"
```

Comment:
Need to add additional distros from https://github.com/deckhouse/deckhouse/blob/main/ee/candi/bashible/detect_bundle.sh